iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -238,5 +238,29 @@ units::Duration CmdLineParser::getProcessKillDelay() const noexcept
     return m_processKillDelay;
 }
 
+iox::log::LogStream& operator<<(iox::log::LogStream& logstream, const MonitoringMode& mode)
+{
+    switch (mode)
+    {
+    case MonitoringMode::OFF:
+        logstream << "OFF";
```

Comment:
I think this is not expressive enough. `ON` and `OFF` could be anything and the user has to rely on the future developer to provide him some context in a log message what on and off means. 

Could you please instead translate it into: `MonitoringMode::ON` and `MonitoringMode::OFF`

Reply:
Fixed.